src/main/java/com/school/project/model/entities/Subject.java

```diff
@@ -1,8 +1,26 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "SUBJECT")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class Subject {
 
+    @Id
+    @Column(name = "SUBJECT_ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @Column(name = "NAME",nullable = false)
```

Comment:
Pls use it: The auto-formatting shortcut in IntelliJ

For windows Ctrl+Alt+L.
For ubuntu Ctrl+Alt+windows+L.
For Mac Command+Option+L.